bower.json

```diff
@@ -17,10 +14,9 @@
     "output",
     "test",
     "bower.json",
-    "gulpfile.js",
     "package.json"
   ],
   "dependencies": {
-    "purescript-prelude": "^0.1.0"
+    "purescript-prelude": "^1.0.0"
```

Comment:
Before I go ahead and open a bunch of other PRs for the libraries I've been working on, are we intending on doing the bump-to-1.0 thing for this round of updates? Or should I continue with the 0.x.x versioning again?